src/app/core/SessionManager.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\core;
+
+class SessionManager
+{
+    public function __construct()
+    {
+        session_start();
+
```

Comment:
Should: TTL of your session id is 180+180=360, use either time()-360 on line 11 or  time()+360 on line 14. No reason for divide it